assignments/this.js

```diff
@@ -10,17 +17,54 @@
 */
 
 // Principle 1
+// ===========code example for Window Binding========
+// method
+// Create an object:
+const person = {
+    firstName: "John",
+    lastName : "Doe",
+    id     : 26,
+    fullName : function() {
+      return this.firstName + " " + this.lastName;
```

Comment:
This is actually Implicit binding because it will bind to `person` object.

You can test it with `console.log(person.fullName());`